### ISSUE TITLE
Additional message

### ISSUE THREAD

**Issue by fwh1990** [typicode/husky#626]
How to dismiss message like
`node_modules/.bin/husky-run pre-commit ''`
 and
`/node_modules/.bin/husky-run prepare-commit-msg '.git/COMMIT_EDITMSG message'`

My configuration is:
```bash
{
  "hooks": {
    "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
  }
}
```

Husky version: 4.0

**Comment by wKovacs64** [comment]
Yeah, what's up with all the extra output now? Happening on Windows 10 (Git Bash) and macOS (zsh). Here's a comparison from macOS:

### Husky v3.1.0

```
$ git commit -m "docs: update README with husky v3
husky > pre-commit (node v12.14.1)
  ↓ Stashing changes... [skipped]
    → No partially staged files found...
  ✔ Running tasks...
husky > commit-msg (node v12.14.1)
[develop 10cdd886] docs: update README with husky v3
 1 file changed, 2 insertions(+)
```

### Husky v4.0.0

```
$ git commit -am "docs: update README with husky v4"
yarn run v1.21.1
$ /Users/user/husky-testing/node_modules/.bin/husky-run pre-commit ''
husky > pre-commit (node v12.14.1)
  ↓ Stashing changes... [skipped]
    → No partially staged files found...
  ✔ Running tasks...
✨  Done in 0.80s.
yarn run v1.21.1
$ /Users/user/husky-testing/node_modules/.bin/husky-run prepare-commit-msg '.git/COMMIT_EDITMSG message'
✨  Done in 0.22s.
yarn run v1.21.1
$ /Users/user/husky-testing/node_modules/.bin/husky-run commit-msg .git/COMMIT_EDITMSG
husky > commit-msg (node v12.14.1)
✨  Done in 0.71s.
yarn run v1.21.1
$ /Users/user/husky-testing/node_modules/.bin/husky-run post-commit ''
✨  Done in 0.21s.
[master 43c8c34a] docs: update README with husky v4
 1 file changed, 2 insertions(+)
```

**Comment by fwh1990** [comment]
It looks strange. I just want to output error message. Current behavior looks like debug output what I don't care

**Comment by fwh1990** [comment]
It looks strange. I just want to output error message. Current behavior looks like debug output what I don't care

**Comment by typicode** [comment]
Thanks for the heads up, such output wasn't really intended. I'll have a look.

**Comment by typicode** [comment]
I just published Husky 4.0.1 which should make things less verbose.
